prime-router/settings/STLTs/OH/oh-doh.yml

```diff
@@ -0,0 +1,216 @@
+          MSH-4:
+            - '*': "CDC PRIME^36DSMP9999"
+          MSH-5:
+            - '*': "OHDOH^2.16.840.1.114222.4.1.3674^ISO"
+          MSH-6:
+            - '*': "OHDOH^2.16.840.1.114222.4.1.3674^ISO"
+        reportingFacilityName: "CDC PRIME"
+        reportingFacilityId: "36DSMP9999"
+        reportingFacilityIdType: null
+        suppressQstForAoe: true
+        suppressHl7Fields: "OBX-23-11, MSH-19-1, MSH-19-2, MSH-19-3"
+        suppressAoe: false
+        defaultAoeToUnknown: false
+        replaceUnicodeWithAscii: false
+        useBlankInsteadOfUnknown: "patient_race"
+        truncateHDNamespaceIds: false
+        truncateHl7Fields: null
+        usePid14ForPatientEmail: false
+        convertTimestampToDateTime: null
+        cliaForOutOfStateTesting: null
+        cliaForSender: {}
+        phoneNumberFormatting: "STANDARD"
+        suppressNonNPI: false
+        processingModeCode: null
+        replaceDiiWithOid: null
+        applyOTCDefault: false
+        useOrderingFacilityName: "STANDARD"
+        valueSetOverrides: {}
+        nameFormat: "OHIO"
+        receivingOrganization: null
+        convertPositiveDateTimeOffsetToNegative: true
+        stripInvalidCharsRegex: null
+        convertDateTimesToReceiverLocalTime: false
+        useHighPrecisionHeaderDateTimeFormat: false
+        type: "HL7"
+        truncationConfig:
+          truncateHDNamespaceIds: false
+          truncateHl7Fields: []
+          customLengthHl7Fields: {}
+      jurisdictionalFilter:
+        - "(Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state = 'OH') or (Bundle.entry.resource.ofType(Patient).address.state = 'OH')"
+      qualityFilter:
+        # Message id is not empty (MSH-10)
+        - "Bundle.identifier.value.exists()"
+        - "Bundle.entry.resource.ofType(Patient).name.family.exists()"
+        - "Bundle.entry.resource.ofType(Patient).name.given.count() > 0"
+        - "Bundle.entry.resource.ofType(Patient).birthDate.exists()"
+        #  Specimen type (SPM-4) is not empty
+        - "Bundle.entry.resource.ofType(Specimen).where(type.empty()).count() = 0"
+        # Patient has at least a street address(PID-11-1), postal code (PID-11-5),phone(PID-13) or email(PID-13-4)
+        - "Bundle.entry.resource.ofType(Patient).where(address.line[0].empty() and address.postalCode.empty() and telecom.where(system = 'phone').value.empty() and telecom.where(system = 'email').value.empty()).count() = 0"
+        - "((Bundle.entry.resource.ofType(Specimen).collection.collectedPeriod.exists() or Bundle.entry.resource.ofType(Specimen).collection.collected.exists())
+                   or (Bundle.entry.resource.ofType(ServiceRequest).occurrence.exists() or Bundle.entry.resource.ofType(Observation).effective.exists()))"
+      routingFilter: []
+      processingModeFilter: []
+      reverseTheQualityFilter: true
```

Comment:
I wonder, how are we going to handle reverseTheQualityFilter: true?

Reply:
I thought this was something that was implemented and working. Is it not @victor-chaparro ?